src/platforms/common/configuration/filtering/index.mdx

```diff
@@ -14,68 +14,47 @@ We also offer [Inbound Filters](/product/data-management-settings/filtering/) to
 
 ## Filtering Error Events
 
-Configure your SDK to filter error events by using the <PlatformIdentifier name="before-send" /> callback method and configuring, enabling, or disabling integrations.
+You can configure your SDK to filter events in a number of ways:
+
+- by using the <PlatformIdentifier name="before-send" /> callback method
+- by using event processors,
+- by configuring, enabling, or disabling integrations.
 
 ### Using <PlatformIdentifier name="before-send" />
 
-All Sentry SDKs support the <PlatformIdentifier name="before-send" /> callback method. <PlatformIdentifier name="before-send" /> is called immediately before the event is sent to the server, so it’s the final place where you can edit its data. It receives the event object as a parameter, so you can use that to modify the event’s data or drop it completely (by returning `null`) based on custom logic and the data available on the event.
+All Sentry SDKs support the <PlatformIdentifier name="before-send" /> callback method. <PlatformIdentifier name="before-send" /> is called immediately before the event is sent to the server, so it's the final place you can edit an event's data. It receives the event object as a parameter, which you can use to modify the event's data or to drop the event completely (by returning `null`) based on custom logic and the data available on the event.
 
 <PlatformContent includePath="configuration/before-send" />
 
 Note also that breadcrumbs can be filtered, as discussed in [our Breadcrumbs documentation](/product/error-monitoring/breadcrumbs/).
 
 <PlatformSection notSupported={["apple", "dotnet"]}>
 
-#### Event Hints
-
-The <PlatformIdentifier name="before-send" /> callback is passed both the `event` and a second argument, `hint`, that holds one or more hints.
-
-Typically a `hint` holds the original exception so that additional data can be extracted or grouping is affected. In this example, the fingerprint is forced to a common value if an exception of a certain type has been caught:
-
-<PlatformContent includePath="configuration/before-send-hint" />
-
-When the SDK creates an event or breadcrumb for transmission, that transmission is typically created from some sort of source object. For instance, an error event is typically created from a log record or exception instance. For better customization, SDKs send these objects to certain callbacks (<PlatformIdentifier name="before-send" />, <PlatformIdentifier name="before-breadcrumb" /> or the event processor system in the SDK).
-
-</PlatformSection>
-
-<PlatformSection notSupported={["native", "apple", "dotnet"]}>
+### Hints
```

Comment:
This section doesn't seem to be about event processors. Is this supposed to undergo more changes?